src/app/app-routing.module.ts

```diff
@@ -4,17 +4,22 @@ import { ProjectDetailComponent } from "./project-tabs/project-detail/project-de
 import { ProjectInvoiceComponent } from "./project-tabs/project-invoice/project-invoice.component";
 import { ProjectResourceComponent } from "./project-tabs/project-resource/project-resource.component";
 import { ProjectStatusComponent } from "./project-tabs/project-status/project-status.component";
+import { ProjectFormComponent } from "./shared/project-form/project-form.component";
+import { ResourceFormComponent } from "./shared/resource-form/resource-form.component";
 
 const appRoutes: Routes = [
-    // { path: '', redirectTo: '/...', pathMatch: 'full'},
-    // { path: '...', component: ...Component, children:[
-    //     { path: ':id', component: RecipeDetailComponent }
-    // ] },
-    { path: 'detail', component: ProjectDetailComponent },
-    { path: 'resource', component: ProjectResourceComponent },
-    { path: 'invoice', component: ProjectInvoiceComponent },
-    { path: 'status', component: ProjectStatusComponent }
-]
+  {path: '', redirectTo: '/detail', pathMatch: 'full'},
+  {path: 'detail', component: ProjectDetailComponent},
+  {path: 'resource', component: ProjectResourceComponent},
+  {path: 'invoice', component: ProjectInvoiceComponent},
+  {path: 'status', component: ProjectStatusComponent},
+  {path: 'projectform', component: ProjectFormComponent, children: [
```

Comment:
Better to set a meaningful path, like for creating a project 'project/create' for editing 'project/edit/{id}'

Reply:
Changed the routing path names.